src/diffusers/loaders/single_file.py

```diff
@@ -147,6 +148,8 @@ def set_additional_components(
                 "force_zeros_for_empty_prompt": False if is_refiner else True,
             }
         )
+    if pipeline_class_name == "StableDiffusionXLInstructPix2PixPipeline":
```

Comment:
Not needed. The kwarg will get passed and set in the pipeline here
https://github.com/huggingface/diffusers/blob/08bf7545070764d41637e90290ca9b92b392263e/src/diffusers/loaders/single_file.py#L317

Reply:
Addressed in 56ad342f